### PR TITLE
feat: try 2 use code.json config, translations website

### PR DESCRIPTION
**Pull request by barchielAbyss**
i want try to try add i18n code.json in website, like docusaurus docs: https://docusaurus.io/docs/i18n/tutorial
Do you have any better ideas, can call to me email

Screenshots of the change:
image

image

### CODE REVIEW

**Comment by juzhiyuan** [comment]
Thanks for your contribution! I just switched the target branch to `master`, and could you please resolve those conflicts first?

File: website/i18n/zh/code.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "hero.webpage.title.fragment2": {
+    "message": "API 信息",
+    "description": "API Traffic"
+  },
+  "hero.webpage.title.fragment3": {
+    "message": "管理",
+    "description": "management"
+  },
+  "hero.webpage.subtitle.content": {
+    "message": "Apache APISIX 提供了丰富的流量管理功能，如负载平衡、Dynamic Upstream、蜡烛版本、代码断路器、身份校验、所见即所得等...",
```
Comment (juzhiyuan) [inline website/i18n/zh/code.json]:
```suggestion
    "message": "Apache APISIX 提供了丰富的流量管理功能，如负载平衡、动态上游、金丝雀发布、断路器、身份校验、可观测性等",
```

File: website/i18n/zh/code.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "hero.webpage.title.fragment1": {
+    "message": "简洁且敏捷的",
```
Comment (juzhiyuan) [inline website/i18n/zh/code.json]:
```suggestion
    "message": "简洁高效的",
```

File: website/i18n/zh/code.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    "description": "Effortless and smooth"
+  },
+  "hero.webpage.title.fragment2": {
+    "message": "API 信息",
```
Comment (juzhiyuan) [inline website/i18n/zh/code.json]:
```suggestion
    "message": "API 流量",
```

File: website/i18n/zh/code.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  "docs.webpage.title.Document": {
+    "message": "文档",
+    "description": "Document"
+  },
+  "docs.webpage.title.DocumentSubtitle": {
+    "message": "开源代表着一种向善的力量",
+    "description": "We love open source."
+  }
+}
```
Comment (juzhiyuan) [inline website/i18n/zh/code.json:34]:
```suggestion
}

```

File: website/i18n/zh/code.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  "arrowAnim.webpage.link.btn": {
+    "message": "阅读文档",
+    "description": "Go to docs..."
+  },
+  "docs.webpage.title.Document": {
+    "message": "文档",
+    "description": "Document"
+  },
+  "docs.webpage.title.DocumentSubtitle": {
+    "message": "开源代表着一种向善的力量",
```
Comment (SylviaBABY) [inline website/i18n/zh/code.json]:
```suggestion
    "message": "让产品与技术背后的细节更加可视化",
```

Comment (SylviaBABY) [reply website/i18n/zh/code.json]: 个人建议：由于这句话是显示在「文档」页面，所以或许加一个解释文档作用的话可能看起来会更好一些。

这里直接放描述开源的话，感觉突然氛围描述很大而且转折看起来有些奇怪。
image

Comment (barchielAbyss) [reply website/i18n/zh/code.json]: 猴

**Comment by barchielAbyss** [comment]
> Thanks for your contribution! I just switched the target branch to `master`, and could you please resolve those conflicts first?

may be i need open the new PR,cherry-pick to my forked warehouse master, i see this commit
chore: test labeler (#1071)  c38f6153335378344595f0456c0cb2acf9449403,
it's can't merge 2 master branch, right?

**Comment by juzhiyuan** [comment]
> > Thanks for your contribution! I just switched the target branch to `master`, and could you please resolve those conflicts first?
> 
> may be i need open the new PR,cherry-pick to my forked warehouse master, i see this commit chore: test labeler (#1071) [c38f615](https://github.com/apache/apisix-website/commit/c38f6153335378344595f0456c0cb2acf9449403), it's can't merge 2 master branch, right?

Ah, cherry-pick is also a good way!

**Pull request closed without merging**